x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -378,6 +395,71 @@ export class TimeSeriesExplorer extends React.Component {
       );
   };
 
+  getPlotByOptions = () => {
+    const { selectedJobId, selectedDetectorIndex } = this.props;
+    const { actualPlotFunction, defaultFunctionToPlotIfMetric } = this.state;
+    this.setPlotByFunction(
+      getPlotByOptions(
+        selectedJobId,
+        selectedDetectorIndex,
+        actualPlotFunction,
+        defaultFunctionToPlotIfMetric
+      )
+    );
+  };
+  /**
+   * Loads available entity values.
+   * @param {Array} entities - Entity controls configuration
+   * @param {Object} searchTerm - Search term for partition, e.g. { partition_field: 'partition' }
+   */
+  loadEntityValues = async (entities, searchTerm = {}) => {
```

Comment:
Does this need removing in favor of the changes made in #81510 ?

Reply:
Deleted here https://github.com/elastic/kibana/pull/81662/commits/a583988eba972e5a1e395aff7c7f613464407e10